schema.proto

```diff
@@ -75,6 +75,12 @@ message RuleV2 {
 
 message CheckV2 {
   repeated RuleV2 queries = 1;
+  option Kind kind = 2;
+
+  enum Kind {
+    None = 0;
```

Comment:
wouldn't `any` be more suited? (not super important since it's only internal, but it's always better to strive for consistency)

Reply:
right, that would make more sense

Reply:
I think I messed up twice when copying, it was "One" not "None"